data/xml/2023.konvens.xml

```diff
@@ -0,0 +1,256 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.konvens">
+  <volume id="main" ingest-date="2024-01-29" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of the 18th Conference on Natural Language Processing (KONVENS 2023)</booktitle>
```

Comment:
19th. Please see [the checklist](https://github.com/acl-org/acl-anthology/wiki/Ingestion-Checklist) which includes checking for inconsistencies on the venue page.

<img width="898" alt="image" src="https://github.com/acl-org/acl-anthology/assets/455256/b6d818ef-b6b4-4227-a6bf-90be00c1aa30">